Dockerfiles/admin/Dockerfile

```diff
@@ -88,7 +88,7 @@ RUN apk add --no-cache \
       openjdk8 \
       bash su-exec openssl tzdata curl jq \
       fontconfig ttf-dejavu ttf-freefont ttf-liberation ttf-linux-libertine \
-      ffmpeg sox hunspell tesseract-ocr \
+      ffmpeg sox hunspell tesseract-ocr nfs-utils \
```

Comment:
Would you mind putting `nfs-utils` on a separat line? I tried to group the packages by category.